### ISSUE TITLE
i965-va-driver breaks xplayer-video-thumbnailer processing mp4/avc/x264 videos on intel graphics.

### ISSUE THREAD

**Issue by smurphos** [linuxmint/xplayer#121]
```
xplayer 1.8.3

System:    Host: steve-HP-Pavilion-g6-Notebook-PC Kernel: 4.15.0-34-generic x86_64
           bits: 64 gcc: 7.3.0
           Desktop: Cinnamon 3.8.9 (Gtk 3.22.30-1ubuntu1)
           Distro: Linux Mint 19 Tara
Machine:   Device: laptop System: Hewlett-Packard product: HP Pavilion g6 Notebook PC v: 0690120000204710000620100 serial: N/A
           Mobo: Hewlett-Packard model: 166F v: 09.4C serial: N/A
           BIOS: Insyde v: F.66 date: 01/24/2013
Battery    BAT0: charge: 36.6 Wh 100.0% condition: 36.6/36.6 Wh (100%)
           model: Hewlett-Packard Primary status: Full
CPU:       Dual core Intel Core i5-2430M (-MT-MCP-) 
           arch: Sandy Bridge rev.7 cache: 3072 KB
           flags: (lm nx sse sse2 sse3 sse4_1 sse4_2 ssse3 vmx) bmips: 9578
           clock speeds: max: 3000 MHz 1: 2625 MHz 2: 2474 MHz 3: 2766 MHz
           4: 2759 MHz
Graphics:  Card: Intel 2nd Generation Core Integrated Graphics Controller
           bus-ID: 00:02.0
           Display Server: x11 (X.Org 1.19.6 )
           drivers: modesetting (unloaded: fbdev,vesa)
           Resolution: 1366x768@60.03hz
           OpenGL: renderer: Mesa DRI Intel Sandybridge Mobile
           version: 3.3 Mesa 18.0.5 Direct Render: Yes
```
Video thumbnails for mp4/avc/x264 are not generated when i965-va-driver is installed. Removing i965-va-driver and deleting the~/.cache/thumbnails resolves the issue. Note the thumbnailer continues to work on xvid/avi files.

Attempting to run xplayer-video-thumbnailer manually on a problem video file with  i965-va-driver installed produces the following output - 

```
xplayer-video-thumbnailer -v "/mnt/backup_drive/Videos/Chromecast Ready/Film/Volver (2006).mp4" test.png
XplayerVideoThumbnailer-Message: 06:58:41.312: Initialised libraries, about to create video widget
XplayerVideoThumbnailer-Message: 06:58:41.316: setting URI file:///mnt/backup_drive/Videos/Chromecast%20Ready/Film/Volver%20(2006).mp4
XplayerVideoThumbnailer-Message: 06:58:41.316: Video widget created
XplayerVideoThumbnailer-Message: 06:58:41.317: About to open video file
XplayerVideoThumbnailer-Message: 06:58:41.527: Checking whether file has cover
XplayerVideoThumbnailer-Message: 06:58:41.528: Opened video file: '/mnt/backup_drive/Videos/Chromecast Ready/Film/Volver (2006).mp4'
XplayerVideoThumbnailer-Message: 06:58:41.528: About to seek to 0.333333
XplayerVideoThumbnailer-Message: 06:58:41.542: About to get frame for iter 0

** (xplayer-video-thumbnailer:2532): WARNING **: 06:58:41.546: Could not take screenshot: failed to retrieve or convert video frame
XplayerVideoThumbnailer-Message: 06:58:41.546: Frame for iter 0 was not interesting
XplayerVideoThumbnailer-Message: 06:58:41.546: About to seek to 0.666667
XplayerVideoThumbnailer-Message: 06:58:41.547: About to get frame for iter 1

** (xplayer-video-thumbnailer:2532): WARNING **: 06:58:41.550: Could not take screenshot: failed to retrieve or convert video frame
XplayerVideoThumbnailer-Message: 06:58:41.550: Frame for iter 1 was not interesting
XplayerVideoThumbnailer-Message: 06:58:41.550: About to seek to 0.100000
XplayerVideoThumbnailer-Message: 06:58:41.552: About to get frame for iter 2

** (xplayer-video-thumbnailer:2532): WARNING **: 06:58:41.555: Could not take screenshot: failed to retrieve or convert video frame
XplayerVideoThumbnailer-Message: 06:58:41.555: Frame for iter 2 was not interesting
XplayerVideoThumbnailer-Message: 06:58:41.555: About to seek to 0.900000
XplayerVideoThumbnailer-Message: 06:58:41.557: About to get frame for iter 3

** (xplayer-video-thumbnailer:2532): WARNING **: 06:58:41.560: Could not take screenshot: failed to retrieve or convert video frame
XplayerVideoThumbnailer-Message: 06:58:41.560: Frame for iter 3 was not interesting
XplayerVideoThumbnailer-Message: 06:58:41.560: About to seek to 0.500000
XplayerVideoThumbnailer-Message: 06:58:41.562: About to get frame for iter 4

** (xplayer-video-thumbnailer:2532): WARNING **: 06:58:41.565: Could not take screenshot: failed to retrieve or convert video frame
XplayerVideoThumbnailer-Message: 06:58:41.565: Frame for iter 4 was not interesting
xplayer-video-thumbnailer couldn't get a picture from '/mnt/backup_drive/Videos/Chromecast Ready/Film/Volver (2006).mp4'
```

**Steps to reproduce**

Install i965-va-driver on appropriate hardware and check thumbnail generation.

**Expected behaviour**

No breakage of the thumbnailer.

**Other information**

**Comment by gm10** [comment]
On LM19 MATE it just fails no matter what, but with different messages depending on whether the va-driver is installed. See my post here:

https://forums.linuxmint.com/viewtopic.php?p=1529928#p1529928

**Comment by smurphos** [comment]
Fixed in Cinnamon 4.0.x presumably by of the 2/10/18 commits. 

Still broken in Cinnamon 3.8.8
